docs/02-database.ipynb

```diff
@@ -0,0 +1,8574 @@
+{
```

Comment:
<p>Explain more about what this is, why you need to define the collection to start.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/eoapi-fedgeoday25-workshop/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>